### PR TITLE
Test mode UI changes

### PR DESCRIPTION
**Pull request by sameer79**
This PR for multiple issues because all are inter-related.
Issue addressed in this PR are below 
1] ISSUE-899
2] ISSUE-902
3] ISSUE-901

### CODE REVIEW

**Comment by arunmahadevan** [comment]
This seem to include the changes from @HeartSaVioR ? If so lets wait until those PRs are merged.

**Comment by sameer79** [comment]
@arunmahadevan  sorry my bad. I missed reverting the backend PR's.
Reverted the commits and updated the PR with proper UI code.
You can merge the backend PR and this one and test out everything.

File: webservice/src/main/resources/app/scripts/components/TopologyGraphComponent.jsx [hunk 1]
```diff
@@ -768,7 +768,7 @@ export default class TopologyGraphComponent extends Component {
           const levelTemp = level > 0 ? level+1 : 1;
           nestedFields(fieldObj[key], levelTemp);
         } else {
-          list.push(`<li style='padding-left : ${level > 0 ? (10*level) : level}px' class=${level > 0 ? 'demo': ''}> ${fieldObj[key]} <span class='output-type'> ${key}</span> </li>`);
+          list.push(`<li style='padding-left : ${level > 0 ? (10*level) : level}px' class=${level > 0 ? 'demo': ''}> ${key} <span class='output-type'> ${fieldObj[key]}</span> </li>`);
```
Comment (shahsank3t) [inline webservice/src/main/resources/app/scripts/components/TopologyGraphComponent.jsx]: We want to keep the styling of the value and field name as it currently is, just replace the position.

File: webservice/src/main/resources/app/scripts/containers/Streams/TestRunComponents/EventLogContainer.jsx [hunk 2]
```diff
@@ -34,18 +34,18 @@ class EventLogContainer extends Component{
...
-        elem.parentNode.removeChild(elem);
-      } else {
-        let eventOverlay = document.createElement('div');
-        eventOverlay.setAttribute('id','eventDiv');
-        eventOverlay.setAttribute('class','eventOverlay');
-        let body = document.getElementsByTagName('body').item(0);
-        body.appendChild(eventOverlay);
-      }
-    }
+    // if(!this.completed){
```
Comment (shahsank3t) [inline webservice/src/main/resources/app/scripts/containers/Streams/TestRunComponents/EventLogContainer.jsx]: We can get rid of this commented code, no need of it now.

File: webservice/src/main/resources/app/scripts/containers/Streams/TopologyEditor/TopologyEditorContainer.jsx [hunk 3]
```diff
@@ -1231,41 +1233,51 @@ class TopologyEditorContainer extends Component {
...
                       });
                       this.setState({eventLogData : events.entities ,hideEventLog :false, testHistory : testHistory,testCompleted : true}, () => {
-                        clearInterval(this.interval);
-                        const msg =  this.state.eventLogData.length ?  "TestRun completed successfully" : "TestRun has No Record" ;
-                        FSReactToastr.success(<strong>{msg}</strong>);
+                        if(testHistory.finished){
+                          this.setState({testRunningMode : false}, () => {
+                            clearInterval(this.interval);
+                            const msg =  this.state.abortTestCase
+                                          ? "Test Run has been Abort successfully"
```
Comment (shahsank3t) [inline webservice/src/main/resources/app/scripts/containers/Streams/TopologyEditor/TopologyEditorContainer.jsx]: Change the msg to "Test Run aborted successfully"

**Comment by shahsank3t** [comment]
@sameer79 Went through the code and left few comments that you might want to address.

**Comment by sameer79** [comment]
@shahsank3t   Address all review comments.

**Comment by sameer79** [comment]
@arunmahadevan  @shahsank3t  Fixed the other UI bugs and integrated killing of test case into UI

**Pull request merged**